test/utils_spec_runner.py

```diff
@@ -427,6 +429,12 @@ def check_events(self, test, listener, session_ids):
                         elif key not in actual:
                             self.fail("Expected key [%s] in %r" % (key, actual))
                         else:
+                            # Workaround an incorrect command started event in fle2v2-CreateCollection.yml
```

Comment:
Is there another ticket tracking this?

Reply:
Ask and ye shall receive: https://jira.mongodb.org/browse/PYTHON-3695